### PR TITLE
Refactor Due To Roles no longer being types

### PR DESCRIPTION
**Pull request by fppt**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -68,7 +68,7 @@ Version v0.8.0
 - Re-add unintentionally deleted test (#772)
 - Fixing quotes in migration.sh (#771)
 - Admin API for Grakn Graphs (#770)
-- roleTypes map bug fixed (#769)
+- roles map bug fixed (#769)
```
Comment (aelred) [inline CHANGELOG]: We shouldn't change the changelog!

File: grakn-core/src/main/java/ai/grakn/util/Schema.java [hunk 2]
```diff
@@ -119,7 +119,7 @@ public static boolean isMetaLabel(TypeLabel label) {
         //Ontology Elements
         ONTOLOGY_ELEMENT(OntologyConcept.class),
         TYPE(Type.class),
-        ROLE_TYPE(RoleType.class),
+        ROLE_TYPE(Role.class),
```
Comment (aelred) [inline grakn-core/src/main/java/ai/grakn/util/Schema.java]: `ROLE_TYPE` -> `ROLE`

File: CHANGELOG (hunk 1 above)
Comment (fppt) [inline CHANGELOG]: oops

File: grakn-core/src/main/java/ai/grakn/util/Schema.java (hunk 2 above)
Comment (fppt) [inline grakn-core/src/main/java/ai/grakn/util/Schema.java]: good catch

**Comment by fppt** [comment]
@aelred re-review please

**Pull request merged**